### PR TITLE
[Feat] [snt9|2_5] Added a case: Resnet101-based Pneumonia recognition

### PR DESCRIPTION
**Pull request by ilovexsir**
## Test Environment
Hardware: 1*ascend-snt9|ARM: 24核 96GB
Image: python3.9-ms2.5.0-cann8.0.0.beta1

## Test Report

Accuracy: 74.1%
Inference Time: 0.065 ms per image
Training Time: 33 minutes
Total Script Runtime: 35 minutes

## Related Issues

Fixes #4

### CODE REVIEW

**Comment by hongziqi** [comment]
`dev/ascend/snt9/2_5/基于Mindspore实现贫血症识别.ipynb` had merge. You should synchronize the main repository first.

**Comment by ilovexsir** [comment]
ok, I do it, please review it.

**Comment by ilovexsir** [comment]
I have a question when I git push:
fatal: unable to access 'http://github.com/ilovexsir/xihe-cases.git/': Failed to connect to github.com port 443 after 21034 ms: Could not connect to server
I try some suggestions by website but It can't solve the problem.
Do you have some suggestions?

**Comment by hongziqi** [comment]
> I have a question when I git push: fatal: unable to access '[http://github.com/ilovexsir/xihe-cases.git/](https://github.com/ilovexsir/xihe-cases.git/)': Failed to connect to github.com port 443 after 21034 ms: Could not connect to server I try some suggestions by website but It can't solve the problem. Do you have some suggestions?

You can clone the repository via SSH, which tends to be more stable than using HTTPS.

**Comment by hongziqi** [comment]
Please rename your filename as 'dev/ascend/snt9/2_5/ResNet101实现肺炎识别.ipynb'

File: dev/ascend/snt9/2_5/基于Mindspore实现肺炎识别.ipynb [hunk 1]
```diff
@@ -0,0 +1,953 @@
...
+    "for epoch in range(num_epochs):\n",
+    "    curr_loss = train(data_loader_train, epoch)\n",
+    "    curr_acc = evaluate(data_loader_val)\n",
+    "    if (epoch + 1) % 10 == 0:\n",
+    "        print(\"-\" * 50)\n",
+    "        print(\n",
+    "            \"Epoch: [%3d/%3d], Average Train Loss: [%5.3f], Accuracy: [%5.3f]\"\n",
+    "            % (epoch + 1, num_epochs, curr_loss, curr_acc)\n",
+    "        )\n",
+    "        print(\"-\" * 50)\n",
```
Comment (hongziqi) [inline dev/ascend/snt9/2_5/基于Mindspore实现肺炎识别.ipynb:843]:
```suggestion
```

**Comment by ilovexsir** [comment]
I update it, please review it.

**Review by hongziqi — APPROVED** [review]

**Comment by hongziqi** [comment]
To maintain a clean Git history, you can squash multiple commits into a single commit:
`git rebase -i HEAD~[N]` # Replace [N] with the number of commits to combine

**Comment by hongziqi** [comment]
Your commits includes contributions from multiple authors, do you need to change？If not I will squash and merge by keeping all Co-authored-by tags as follows:
```
---------

Co-authored-by: ilovescrapy <2623969821@qq.com>
Co-authored-by: ilovexsir <xsir156@163.com>
```

**Comment by ilovexsir** [comment]
Both are my accounts, but the first one was restricted by github, please use ilovexsir <xsir156@163.com>

File: dev/ascend/snt9/2_5/ResNet101实现肺炎识别.ipynb [hunk 2]
```diff
@@ -0,0 +1,942 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fa7e3e52",
+   "metadata": {},
+   "source": [
+    "# ResNet101实现肺炎识别\n",
+    "\n",
+    "本项目是通过检测X片的肺部图片，检查是否有肺部炎症，该项目能够辅助医生更快的做出判别，减少医生的工作量。\n"
```
Comment (hongziqi) [inline dev/ascend/snt9/2_5/ResNet101实现肺炎识别.ipynb:10]: If there are ​more than three categories, it is recommended to explicitly list all category names.

Comment (ilovexsir) [reply dev/ascend/snt9/2_5/ResNet101实现肺炎识别.ipynb]: OK，I update it.

**Review by hongziqi — APPROVED** [review]
